omnibus/config/software/rpm.rb

```diff
@@ -22,14 +22,9 @@
 skip_transitive_dependency_licensing true
 
 dependency "config_guess"
-dependency "elfutils"
 dependency "file"
-dependency "libgpg-error"
-dependency "libgcrypt"
 dependency "popt"
 dependency "zstd"
-dependency "libsqlite3"
-dependency "libdb"
```

Comment:
I'm curious how did you do manage to shrink it down? Looking at what's actually used downstream, relying on compilation/linking errors, comparing to `deb` deps, or something else?

Reply:
Mostly looking at their build system & docs to see what we could reasonably disable
* elfutils is optional and only appears to be used when creating a package
* gcrypt/gpg-error shouldn't be needed when relying on the openssl crypto backend
* libsqlite appears to be overzealous after discussing with @0intro
* libdb seems unneeded, according to https://github.com/rpm-software-management/rpm/blob/master/INSTALL#L87C2-L89

Reply:
Can we also get rid of `lua`?